### PR TITLE
Cleanup ordering a bit, create atomic commit of bulk data.

### PR DESCRIPTION
**Pull request by morrowc**
Cleanup some ordering of var/const/etc.
Create a single transaction for the bulk insert event.
This does two things:
  1) pay for statement creation 1x
  2) pay for indexing costs 1x
  3) make sure the data on the user facing portion is consistent
     (we haven't inserted half the data when the user requests content)

### CODE REVIEW

File: arcgis_scraper/arcgis_scraper.go [hunk 1]
```diff
@@ -4,24 +4,32 @@ import (
...
 	"encoding/json"
+	"fmt"
 	"net/http"
 	"time"
-	"fmt"
+
 	"github.com/jinzhu/configor"
 
 	"github.com/gidoBOSSftw5731/log"
+	"github.com/lib/pq"
```
Comment (gidoBOSSftw5731) [inline arcgis_scraper/arcgis_scraper.go]: Why import it this way instead of with the _?

Comment (morrowc) [reply arcgis_scraper/arcgis_scraper.go]: unclear! I don't think I atually changed that prboably that's the work of gofmt ;(

Comment (morrowc) [reply arcgis_scraper/arcgis_scraper.go]: oh! because I'm using a pq function. so yea, nthis is necessary.

**Comment by gidoBOSSftw5731** [comment]
I'll test this locally before pushing, and yes, I know I should do this differently with real tests, but that's fancy pants

File: arcgis_scraper/arcgis_scraper.go [hunk 2]
```diff
@@ -14,14 +14,23 @@ import (
...
-		IP       string `default:"127.0.0.1"`
-	}
-}{}
+
+const (
+	arcgisURL = "https://opendata.arcgis.com/datasets/628578697fb24d8ea4c32fa0c5ae1843_0.geojson"
+)
+
+var (
+	db     = *sql.DB
```
Comment (gidoBOSSftw5731) [inline arcgis_scraper/arcgis_scraper.go]: Causes error "./arcgis_scraper.go:23:11: type *sql.DB is not an expression"

**Pull request merged**